handout/features/es6_modules.md

```diff
@@ -13,13 +13,7 @@ export const pi = 3.141592;
 export const circumference = diameter => diameter * pi;
 ```
 
-The code above uses the _Arrow_ function for `circumference`, which was introduced in ES6, and is a shortform for the following.
-
-```js
-export function circumference(diameter) {
-  return diameter * pi;
-}
-```
+The code above uses the _Arrow_ function for `circumference`. Read more about arrow functions [here](https://angular-2-training-book.rangle.io/handout/features/arrow_functions.html)
```

Comment:
This is my recommendation for the paragraph:

"The code above uses an _arrow_ function for `circumference` which was introduced in ES6. To learn more about, visit the [Arrow Function](./arrow_functions.md) section in the book."

Why?

- We should make obvious to the user that it's an internal link, not external.
- We should use relative links pointing to the markdown file so we can use the book in a dev environment as well.